Rubberduck.RibbonDispatcher/Abstract/IRibbonCommon.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+using stdole;
+using Microsoft.Office.Core;
+using System.Runtime.InteropServices;
+
+using Rubberduck.RibbonDispatcher;
+
+namespace Rubberduck.RibbonDispatcher.Abstract
+{
+    using ControlSize         = RibbonControlSize;
+    using ChangedEventHandler = EventHandler<ChangedControlEventArgs>;
```

Comment:
scoped using statements, which aren't used anywhere else in the codebase

Reply:
I have been almost the only person reading my code for longer than C# has been around - so undoubtedly my style is idiosyncratic. I just developed the habit of putting the library usings up-top, out of the way, and the interesting ones closer to the action. Having the namespace between them is just a means of conveniently separating the one from the other. But just a habit, really.